joomla/joomla-cms
# Retain filtered month in archive form

This is a partial fix for Issue #19382.

### Summary of Changes
In the Archive search form, the month dropdown has leading zero from Jan. to Sep. 
```
<select id="month" name="month" size="1" class="inputbox">
	<option value="">Month</option>
	<option value="01">Jan</option>
	<option value="02">Feb</option>
	<option value="03">Mar</option>
	<option value="04">Apr</option>
	<option value="05">May</option>
	<option value="06">Jun</option>
	<option value="07">Jul</option>
	<option value="08">Aug</option>
	<option value="09">Sep</option>
	<option value="10">Oct</option>
	<option value="11">Nov</option>
	<option value="12">Dec</option>
</select>
```

This PR removes the leading zero to match the month query string so the chosen month is selected: 
`http://localhost/joomla-cms-staging/archived-articles/2011?view=archive&month=1`


### Testing Instructions
Install demo.
Click `Archive` link.
Click `January, 2011`.
In the archive form, see the month dropdown.


### Expected result
`Jan` is selected.


### Actual result
`Month` is selected.


### Documentation Changes Required
none
